hoster/utils/fileutils.go

```diff
@@ -0,0 +1,77 @@
+package utils
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"regexp"
+)
+
+type IFileUtils interface {
+	InjectRule(envRulesPath string) error
+	ClearRule(envRulesPat string) error
+	ClearAll()
+}
+
+type fileUtils struct {
+	envFile string
+}
+
+const hosterHostsBlockPrefix string = "^\\# Added by Hoster"
+
+const hosterBlock string = `
+# Added by Hoster
+# End of section
+`
+
+func (fu *fileUtils) EnsureHosterBlock() (err error) {
```

Comment:
what is this `fu` in here? :D

Reply:
This part makes sure a proper etc hosts file structure exists so hoster can act on it later in the process.